enumeration/remote/aws/repository/s3control_repository.go

```diff
@@ -0,0 +1,47 @@
+package repository
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3control"
+	"github.com/snyk/driftctl/enumeration/remote/aws/client"
+	"github.com/snyk/driftctl/enumeration/remote/cache"
+)
+
+type S3ControlRepository interface {
+	DescribeAccountPublicAccessBlock() (*s3control.PublicAccessBlockConfiguration, error)
+}
+
+type s3ControlRepository struct {
+	clientFactory client.AwsClientFactoryInterface
+	accountId     string
+	cache         cache.Cache
+}
+
+func NewS3ControlRepository(factory client.AwsClientFactoryInterface, c cache.Cache) *s3ControlRepository {
+	return &s3ControlRepository{
+		clientFactory: factory,
+		cache:         c,
+	}
+}
+func (s *s3ControlRepository) GetAccountID() string {
```

Comment:
Is this method still used ?

Reply:
yeah sorry I pushed an half done thing for you to comment on where the accountid was injected. It's fixed now.